### DISCUSSION TITLE
Start container error

### DISCUSSION THREAD

**Discussion by axil76** [open-webui/open-webui#1291] [Q&A]
Hello,
I launched the creation of the containers and I have the open-webui container which tells me that the python module is not found 
File "/app/backend/main.py", line 18, in <module>
    from litellm.proxy.proxy_server import ProxyConfig, initialize
ImportError: cannot import name 'ProxyConfig' from 'litellm.proxy.proxy_server' (/usr/local/lib/python3.11/site-packages/litellm/proxy/proxy_server.py)
I have the module in the requirement file

Thanks for your help

**Comment by tjbck** [comment]
Ensure your LiteLLM version is set to `litellm==1.30.7`.
